doc/jaas/move-to-jaas.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+PRIVATE_KEY=$1
+JAAS_KEY_ID=$2
+
+if [ ! -f "${PRIVATE_KEY}" ] ; then
+  echo "You need to specify a correct path for the private key as a first argument."
+  exit 1;
+fi
+
+if [[ ! "${JAAS_KEY_ID}" =~ ^vpaas-magic-cookie-[0-9a-z]+/[0-9a-z]+$ ]]; then
+    echo "Invalid key id passed as a second argument."
+    exit 2;
+fi
+
+NODE_VER=$(node -v);
+NODE_MAJOR_VER=$(echo ${NODE_VER:1} |  cut -d. -f1);
+
+echo "$NODE_MAJOR_VER"
+
+if [ "$NODE_MAJOR_VER" -lt "16" ]; then
+    echo "Please install latest LTS version of node (16+)";
+    exit 3;
+fi
+
+# we need this util for debconf-set-selections
+sudo apt install debconf-utils
+
+# Let's pre-set some settings for token-generator
+cat << EOF | sudo debconf-set-selections
+token-generator token-generator/private-key string ${PRIVATE_KEY}
```

Comment:
Well this is used only for debian to be able to pass which private key to use, so it can put it in the config file, no more than that. So it is already the case for the portability.

Reply:
Cool!